data/human/ships.txt

```diff
@@ -3245,7 +3249,7 @@ ship "Star Queen"
 		over
 	gun 0 -113
 	gun -24 -76 "Sidewinder Missile Launcher"
-	gun 24 -76 "Sidewinder Missile Launcher"
+	gun 24 -76
```

Comment:
I wonder if it might not make more sense for the Star Queen to have two Sidewinder Missile Pods rather than a single launcher.

What makes more sense? A quick barrage of missiles to keep an attacker busy/distracted/knocked off long enough to hyperjump? Or a long steady stream of missiles to continually pester them while the Star Queen tries to flee? I think the former - given that we don't need to go anywhere in order to hyperjump, they just need an impact-free moment to jump; they don't need to continually harass anyone while they try to just fly away somewhere.

Reply:
Pods *and extra ammo*.